core/jazz_admin/test/test.js

```diff
@@ -1,11 +1,171 @@
-const assert = require('chai').assert;
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const request = require('request');
+
 const index = require('../index');
+const awsContext = require('aws-lambda-mock-context');
+const configObj = require('../components/config.js');
+
+describe('jazz_admin', function () {
+
+  beforeEach(function () {
+    spy = sinon.spy();

```

Comment:
why do you need this? r u using it anywhere?

Reply:
removed. I had initially thought I needed it but forgot to remove it when I didn't need it.